protocol.md

```diff
@@ -397,14 +395,23 @@ This extension defines how to upload finite streams of data that have an
 unknown length at the beginning of the upload.
 
 If the [Creation](#creation) extension is used to initiate a new upload the
-`Upload-Length` header MUST be set to the string `streaming`. Once the total size of the
-entire upload is known it MUST be included as the `Upload-Length` header's value
-in the next `PATCH` request. Once the upload's length has been set it MUST NOT
-be changed.
+`Upload-Stream` header MUST be set to 1. Once the total size of the entire 
+upload is known `Upload-Length` MUST be set in the next `PATCH` request. 
+`Upload-Length` once set SHALL NOT be changed.
 
 In order to indicate that this extension is supported by the Server it MUST
 include the `stream` element in the `Tus-Extension` header.
 
+#### Headers
+
+##### Upload-Stream
+
+The `Upload-Stream: 1` MUST be included with the file creation request.
+
+##### Upload-Length
+
+The `Upload-Length` MUST be set only once with the `PATCH` request.
```

Comment:
I think the client can sent as many as it wants, but the server only accepts the first one? That will make implementations with parallel running workers easier on the client side